src/s1reader/utils/plot_bursts.py

```diff
@@ -0,0 +1,159 @@
+import argparse
+import glob
+from s1reader.s1_orbit import get_orbit_file_from_list
+from s1reader.s1_reader import load_bursts
+from osgeo import gdal, osr
+from shapely.geometry import Polygon
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import folium
+from shapely import wkt
+import fiona
```

Comment:
```suggestion
import argparse
import glob
from importlib import import_module
import sys

named_libs = [('fiona', 'fiona'), ('folium', 'folium'), ('geopandas', 'gpd'),
            ('pandas', 'pd')]
for (name, short) in named_libs:
    try:
        lib = import_module(name)
    except:
        print(sys.exc_info())
    else:
        globals()[short] = lib

import matplotlib.pyplot as plt
from osgeo import gdal, osr
from shapely.geometry import Polygon
from shapely import wkt

from s1reader.s1_orbit import get_orbit_file_from_list
from s1reader.s1_reader import load_bursts
```
[pep8 import order](https://peps.python.org/pep-0008/#imports)

[error handling multiple imports](https://stackoverflow.com/a/44829497)

Reply:
Please add lines in between import groups for readability. From pep8 import section linked above:

>
> Imports should be grouped in the following order:
> 
>   1. Standard library imports.
>   2. Related third party imports.
>   3. Local application/library specific imports.
> 
> **You should put a blank line between each group of imports.**
>